packages/apps/dashboard/server/src/modules/networks/networks.service.ts

```diff
@@ -0,0 +1,90 @@
+import { Inject, Injectable, Logger } from '@nestjs/common';
+import { ChainId, NETWORKS, StatisticsClient } from '@human-protocol/sdk';
+
+import {
+  EnvironmentConfigService,
+  MINIMUM_ESCROWS_COUNT,
+  MINIMUM_HMT_TRANSFERS,
+} from '../../common/config/env-config.service';
+import { AVAILABLE_NETWORKS_CACHE_KEY } from 'src/common/config/redis-config.service';
+import { Cache, CACHE_MANAGER } from '@nestjs/cache-manager';
+import { MainnetsId } from 'src/common/utils/constants';
+
+@Injectable()
+export class NetworksService {
+  private readonly logger = new Logger(NetworksService.name);
+  constructor(
+    @Inject(CACHE_MANAGER) private cacheManager: Cache,
+    private readonly envConfigService: EnvironmentConfigService,
+  ) {}
+
+  public async getOperatingNetworks(): Promise<ChainId[]> {
+    const cachedNetworks = await this.cacheManager.get<ChainId[]>(
+      AVAILABLE_NETWORKS_CACHE_KEY,
+    );
+
+    if (cachedNetworks) {
+      return cachedNetworks;
+    }
+
+    const currentMonth = new Date();
+    const oneMonthAgo = new Date(currentMonth);
+    oneMonthAgo.setMonth(oneMonthAgo.getMonth() - 1);
+
+    const filterDate = new Date(currentMonth);
+    filterDate.setMonth(
+      filterDate.getMonth() - this.envConfigService.networkUsageFilterMonths,
+    );
+
+    const availableNetworks = [];
+
+    for (const networkKey of Object.values(MainnetsId)) {
+      const chainId = MainnetsId[networkKey as keyof typeof MainnetsId];
+
+      const networkConfig = NETWORKS[chainId];
+
+      if (!networkConfig) {
+        continue;
+      }
+
+      const statisticsClient = new StatisticsClient(networkConfig);
+
+      try {
+        const hmtData = await statisticsClient.getHMTDailyData({
+          from: new Date(Math.floor(filterDate.getTime() / 1000) * 1000),
+        });
+        const escrowStats = await statisticsClient.getEscrowStatistics({
+          from: new Date(Math.floor(oneMonthAgo.getTime() / 1000) * 1000),
+        });
+
+        // Calculate total HMT transaction count across the period
+        const totalTransactionCount = hmtData.reduce(
+          (sum, day) => sum + day.totalTransactionCount,
+          0,
+        );
+
+        // At least 1 escrow created in the last month
+        const recentEscrowsCreated =
+          escrowStats.totalEscrows >= MINIMUM_ESCROWS_COUNT;
+        // Total HMT transactions > MINIMUM_HMT_TRANSFERS in the last X months
+        const sufficientHMTTransfers =
+          totalTransactionCount > MINIMUM_HMT_TRANSFERS;
+
+        if (recentEscrowsCreated && sufficientHMTTransfers) {
+          availableNetworks.push(chainId);
+        }
+      } catch (error) {
+        this.logger.error(
+          `Error processing network ${networkKey} (Chain ID: ${chainId}): ${error.message}`,
+        );
+      }
+    }
+
+    await this.cacheManager.set(
+      AVAILABLE_NETWORKS_CACHE_KEY,
+      availableNetworks,
+      this.envConfigService.networkAvailableCacheTtl,
```

Comment:
Let's rename `networkAvailableCacheTtl` and it's env counterpart to have `operating` word